cmd/tempo-serverless/lambda/Dockerfile

```diff
@@ -34,12 +34,11 @@ COPY --from=build /lambda-local-proxy /lambda-local-proxy
 RUN chmod +x /lambda-local-proxy
 
 RUN echo -e '\
-#!/bin/bash \n\
-AWS_REGION=dummy AWS_ACCESS_KEY_ID=dummy AWS_SECRET_ACCESS_KEY=dummy /lambda-local-proxy -e http://localhost:8080 -p 9000 -f function & \n\
-exec /lambda-entrypoint.sh lambda \n\
-' > /run.sh
+  #!/bin/bash \n\
```

Comment:
This diverges somewhat from the golang example here: https://docs.aws.amazon.com/lambda/latest/dg/go-image.html

But the local proxy could be useful in e2e testing and local development.